tensorboard/plugins/graph/standalone/index.html

```diff
@@ -0,0 +1,71 @@
+<!doctype html>
+<!--
+@license
+Copyright 2016 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<script src="../../webcomponentsjs/webcomponents-lite.min.js"></script>
+<link rel="import" href="../tf-graph-controls/tf-graph-controls.html">
+<link rel="import" href="../tf-graph-loader/tf-graph-loader.html">
+<link rel="import" href="../tf-graph-board/tf-graph-board.html">
+<title>TF Graph Standalone</title>
+<dom-module id="tf-graph-standalone">
```

Comment:
I believe the existing `<tf-graph-app>` component here is supposed to be usable for this purpose:
https://github.com/tensorflow/tensorboard/blob/1.10.0/tensorboard/plugins/graph/tf_graph_app/tf-graph-app.html
https://github.com/tensorflow/tensorboard/blob/1.10.0/tensorboard/plugins/graph/tf_graph_app/index.html

Would it be possible for you to use that existing one, potentially with fixes if you need them?

Reply:
When I bazel run tensorboard/plugins/graph/tf_graph_app, and
a) open tf-graph-app.html: I get an empty page
b) open index.html: I get a page describing how to embed the app with a given pbtxt
From that experience I assumed that was not actually meant to be run just so a user could press a button to load a pbtxt :)

Reply:
I can run the /demo there, and that gives me the demo graph, but there the 'Choose File' button is not connected. I could fix that if that makes more sense...

Reply:
Fixed the demo app in the other patch, abandoning this one.